.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,6 +26,14 @@ assignees:
 
 ```
 
+##### COLLECTION VERSION
+<!--- Paste verbatim output from "ansible-galaxy collection list | grep <collection>"  between the quotes
+for example: ansible-galaxy collection list | grep community.general
```

Comment:
```suggestion
for example: ansible-galaxy collection list community.general
```

No grep required, I learned it last week -- for example:
```
> ansible-galaxy collection list community.general

# /home/dmsimard/dev/virtualenvs/ansible/lib/python3.9/site-packages/ansible_collections
Collection        Version
----------------- -------
community.general 2.0.1  

# /home/dmsimard/dev/virtualenvs/ansible/lib64/python3.9/site-packages/ansible_collections
Collection        Version
----------------- -------
community.general 2.0.1  
```

Reply:
Er, I guess I should have included the comment above in my suggestion. Let me try that again.